### DISCUSSION TITLE
How can i change the time format from 12pm to 24hs

### DISCUSSION THREAD

**Discussion by Chaziz** [rejetto/hfs#813] [Q&A] [answered]
![image](https://github.com/user-attachments/assets/def8b092-1505-485b-b2b3-0c54d4c718de)
I want to change the hour format. How can i do that? i don't want am pm format

**Comment by rejetto** [comment, marked as answer]
that time format is left to the browser, currently, so it's not controlled by HFS configuration but by the browser's configuration.
Most of times, the browser defaults to the operating system.

Based on your screenshot, it appears that HFS is using a Spanish translation, but the time format is American. This suggests that your browser might be set to a Latin American language.

**Comment by Chaziz** [comment]
Tried Argentina and other countries with no luck. Spanish from Spain solved the issue. Thanks!

**Comment by rejetto** [comment]
ok. i was not suggesting to set it to a latin american country, but that it was already so, and that was causing the problem.
so it's normal that Argentina was not good for you.
can you tell me what was the original setting that caused the problem?
